x-pack/plugin/geoip-enterprise-downloader/src/main/java/org/elasticsearch/xpack/geoip/EnterpriseGeoIpDownloaderLicenseListener.java

```diff
@@ -137,6 +152,14 @@ private void ensureTaskStopped() {
                 }
             }
         );
-        persistentTasksService.sendRemoveRequest(ENTERPRISE_GEOIP_DOWNLOADER, MasterNodeRequest.INFINITE_MASTER_NODE_TIMEOUT, listener);
+        persistentTasksService.sendRemoveRequest(
```

Comment:
I think we want to make use of the new method you added, right?
```suggestion
        persistentTasksService.sendProjectRemoveRequest(
        	projectId,
```